src/foam/u2/property/MDSelect.js

```diff
@@ -47,7 +47,8 @@ foam.CLASS({
             choices$: this.choices$,
             index$: this.index$
           });
-          this.popup.open(this.index, this.el());
+          var el = await this.el();
```

Comment:
By the same logic as above, I think we could use el_(), but you could also just leave it as is if you prefer.